apps/src/sites/code.org/pages/views/swiper_farsi_page_videos.js

```diff
@@ -0,0 +1,49 @@
+// Adds carousel functionality to code.org/videos
```

Comment:
Nit: can you update this comment: 
```suggestion
// Adds carousel functionality to code.org/global/fa/videos
```